quick_test.py

```
@@ -0,0 +1,123 @@
+            "text": "Meeting at 3pm today. Please confirm your attendance.",
+            "expected": "legitimate"
+        },
+        {
+            "text": "URGENT: Your UPI account is suspended! Verify immediately: suspicious-link.com",
+            "expected": "phishing"
+        }
+    ]
+    
+    for i, case in enumerate(test_cases, 1):
+        try:
+            response = requests.post(url, json={"text": case["text"]})
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)